fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatService.java

```diff
@@ -50,6 +50,18 @@ Response hasAuthorization(
       @Path("resourceName") String resourceName,
       @Path("authorization") String authorization);
 
+  /**
+   * @param userId The username of the user
+   * @param resourceType The type of the resource
+   * @param resource The resource to check
+   * @return 200 if the user can create the specified resource, 404 otherwise
+   */
+  @POST("/authorize/{userId}/{resourceType}")
+  Response canCreate(
```

Comment:
```suggestion
  void canCreate(
```
this can be void instead of Response - we only care about the Response if we got a RetrofitError and in that case we can get it from there